python/spacewalk/spacewalk-backend.changes.rtp.spacewalk-data-fsck-srpm

```diff
@@ -0,0 +1 @@
+- Fix spacewalk-data-fsck src.rpm handling
```

Comment:
Thanks for the changes.

Just an small note here: this file should be named `spacewalk-backend.changes.apatard.spacewalk-data-fsck-srpm` so it contains your username as part of the filename.

Other than that, it looks good!

Reply:
Ok. I've updated the commit to fix that.